### PR TITLE
Updated MHV resource and spotlight links in landing page

### PR DESCRIPTION
**Pull request by carlosfelix2**
## Summary
This PR updates a few landing page links that are part of the MHV portal. Most of the MHV portal links looked at the SSOe session to provide authenticated links were possible, but some links were not using this check.

## Related issue(s)
https://app.zenhub.com/workspaces/mhv-on-vagov-landing-page-62619a987d74510018ecc546/issues/gh/department-of-veterans-affairs/va.gov-team/77358

## Testing done
- Manual testing

## Screenshots
No changes to the UI.

## What areas of the site does it impact?
- MHV landing page

## Acceptance criteria
- A veteran is taken to the MHV portal authenticated pages when accessing an MHV article from the MHV landing page.

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [x] Events are being sent to the appropriate logging solution
- [x] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

### CODE REVIEW

File: src/applications/mhv-landing-page/utilities/data/index.js [hunk 1]
```diff
@@ -248,19 +248,22 @@ const resolveLandingPageLinks = (
       {
```
Comment (radavis) [inline src/applications/mhv-landing-page/utilities/data/index.js:248]: You probably want to remove the comment on line 245.

**Review by radavis — CHANGES_REQUESTED** [review]
Remove or update comment to reflect code change.

File: src/applications/mhv-landing-page/utilities/data/index.js (hunk 1 above)
Comment (carlosfelix2) [reply src/applications/mhv-landing-page/utilities/data/index.js:248]: Good catch

**Review by radavis — APPROVED** [review]

**Review by rmessina1010 — APPROVED** [review]
LGTM

**Pull request merged**